tests/api/testdata/courses/assignments/submit/courses_assignments_submissions_submit_base.json

```diff
@@ -1,6 +1,7 @@
 {
     "module": "autograder.api.courses.assignments.submissions.submit",
     "arguments": {
+        "allow-late": false
```

Comment:
Could this be left off?
Or is is one of the strange parameter issues where we need to for other test output.

Reply:
It's the param issue where the CLI can't find the correct key in among the API keys. I believe it is because the CLI sets a default value.